AdyenComponents/BACS Direct Debit/Scenes/Input/BACSInputFormViewController.swift

```diff
@@ -0,0 +1,39 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import UIKit
+
+internal protocol BACSInputFormViewProtocol: FormViewProtocol {}
+
+internal class BACSInputFormViewController: FormViewController, BACSInputFormViewProtocol {
```

Comment:
I thought we decided to removed `BACSInputFormViewController` and `BACSConfirmationViewController`?

Reply:
We decided to do the change after the release.

Reply:
👍